design/767-reporting-crmpack-objects/crmPack-documentation-demo-v2.qmd

```diff
@@ -161,17 +173,22 @@ toxModel
 knit_print(toxModel, use_values = FALSE)
 ```
 
-```{r}
-# trialStopping
-# trialNextBest
+The behaviour with nested objects is not as I would have expected, and more work is needed.
```

Comment:
is this specific to stopping rules? or lists?
maybe good to mention here more details